crawl-ref/source/mon-cast.cc

```diff
@@ -472,6 +474,11 @@ static const map<spell_type, mons_spell_logic> spell_to_logic = {
         [] (const monster &caster, mon_spell_slot /*slot*/, bolt& /*beem*/) {
             _monster_abjuration(caster, true);
         }, nullptr, MSPELL_LOGIC_NONE, 20, } },
+    { SPELL_CORRUPT_LOCALE, {
+        _always_worthwhile
```

Comment:
Perhaps consider banning it in abyss, unless you really enjoy the cantrip behaviour :)

Reply:
I've made it a bad() idea when in the Abyss. Not quite banning it, as I assume even bad ideas are picked occasionally. The idea of Mlio pausing to bask in the beautiful corruption of the Abyss occasionally on the off chance he ends up back there seems nice.